server/git/git.go

```diff
@@ -83,12 +83,12 @@ func RunGit(ctx context.Context, in io.Reader, out io.Writer, er io.Writer, dir
 	logger := log.FromContext(ctx).WithPrefix("rungit")
 	c := exec.CommandContext(ctx, "git", args...)
 	c.Dir = dir
-	c.Env = append(c.Env, envs...)
+	c.Env = append(os.Environ(), envs...)
 	c.Env = append(c.Env, "PATH="+os.Getenv("PATH"))
 	c.Env = append(c.Env, "SOFT_SERVE_DEBUG="+os.Getenv("SOFT_SERVE_DEBUG"))
 	if cfg != nil {
-		c.Env = append(c.Env, "SOFT_SERVE_LOG_FORMAT="+cfg.LogFormat)
-		c.Env = append(c.Env, "SOFT_SERVE_LOG_TIME_FORMAT="+cfg.LogTimeFormat)
```

Comment:
q: why do we pass those envs down to `git`?

Reply:
These envs get passed down to `git-receive-pack` and then to git-hooks like `pre-recieve` and `update`. Then in git-hooks, we call the soft-serve binary to run the hooks inside soft-serve.

Reply:
Ahh gotcha! Thanks for explaining 🙏